kmir/src/kmir/__main__.py

```diff
@@ -45,6 +47,25 @@ def __init__(self, input_file: Path, output_file: str | None, start_symbol: str)
         self.start_symbol = start_symbol
 
 
+@dataclass
+class ProveOpts(KMirOpts):
+    spec_file: Path
+    proof_dir: Path | None
+    include_labels: tuple[str, ...] | None
+    exclude_labels: tuple[str, ...] | None
+
+    def __init__(
+        self, spec_file: Path, proof_dir: Path | str | None, include_labels: str | None, exclude_labels: str | None
+    ) -> None:
+        self.spec_file = spec_file
+        if proof_dir is None:
+            self.proof_dir = None
+        else:
+            self.proof_dir = Path(proof_dir).resolve()
```

Comment:
I think this can be the same format as the lines below no?
```suggestion
        self.proof_dir = Path(proof_dir).resolve() if proof_dir is not None else None
```